src/main/resources/locales/zh-CN.yml

```diff
@@ -1,286 +1,282 @@
       ISLAND_RESPAWN:
         description: |-
-          ＆a玩家重生
-          和他们的洞穴
-        name: 洞穴重生
+          &a玩家在他们的
+          &a洞穴中重生
+        name: "洞穴中重生"
       LIQUIDS_FLOWING_OUT:
-        name: 液体在洞穴外流动
+        name: "流体流出洞穴"
         description: |-
-          ＆a切换液体是否可以流出
-          ＆a洞穴的保护范围。
+          &a切换流体是否能
+          &a流出洞穴保护范围.
       LOCK:
-        description: 拨动锁
-        name: 锁洞
+        description: "切换是否禁止访客访问"
+        name: "锁定洞穴"
       NATURAL_SPAWNING_OUTSIDE_RANGE:
-        name: 自然生物产生范围外
+        name: "常规生物在保护范围外生成"
         description: |-
-          ＆a切换是否生物（动物和
-          ＆amonsters）可以在外面自然产卵
-          ＆aa洞穴的保护范围。
+          &a切换常规生物 (动物和
+          &a怪物) 是否能在洞穴保护
+          &a范围外生成.
 
-          ＆c请注意，这不会阻止生物
-          ＆cto通过暴民生成器或生成器生成
-          ＆cegg。
+          &c注意, 此操作不能防止
+          &c洞穴保护范围外的
+          &c刷怪笼或刷怪蛋生成生物.
       OFFLINE_GROWTH:
         description: |-
-          ＆a禁用时，植物
-          ＆a不会在洞穴中生长
-          ＆awhen所有成员均处于离线状态。
-          ＆a可以帮助减少延迟。
-        name: 离线增长
+          &a如果禁用, 当洞穴中的所有成员
+          &a都离线时, 植物将停止生长.
+          &a或许能减少服务器卡顿.
+        name: "离线生长"
       OFFLINE_REDSTONE:
         description: |-
-          ＆a禁用时，红石
-          ＆a不会在山洞中运作
-          ＆a所有成员均处于离线状态。
-          ＆a可以帮助减少延迟。
-        name: 离线红石
+          &a如果禁用, 当洞穴中的所有成员
+          &a都离线时, 红石将停止运作.
+          &a或许能减少服务器卡顿.
+        name: "离线红石"
       PISTON_PUSH:
         description: |-
-          ＆a允许活塞推动
-          ＆在洞穴外面
-        name: 活塞推
+          &a允许活塞推动
+          &a洞穴外的方块
+        name: "活塞推动"
       REMOVE_MOBS:
         description: |-
-          ＆a移除怪物
-          ＆前往洞穴
-        name: 移除怪物
+          &a传送到洞穴中时,
+          &a移除洞穴中的所有怪物
+        name: "Remove monsters"
       TREES_GROWING_OUTSIDE_RANGE:
-        name: 树木生长范围外
+        name: "树木在洞穴保护范围外生长"
         description: |-
-          ＆a切换树木是否可以在
-          是否使用＆acave的保护范围。
+          &a切换树木是否能长到
+          &a洞穴保护范围之外.
       PREVENT_TELEPORT_WHEN_FALLING:
-        name: 跌倒时防止传送
+        name: "掉落时禁止传送"
         description: |-
-          ＆a防止玩家被传送
-          使用命令返回他们的洞穴
-          ＆aif他们跌倒了。
-        hint: "＆c跌倒时无法传送回洞穴。"
-    locked: "＆c这个洞穴被锁了！"
-    protected: "＆cCave保护：[说明]"
-    spawn-protected: "＆cSpawn受保护：[描述]"
+          &a当玩家正在掉落时
+          &a禁止他们通过指令进行传送.
+        hint: "&c在掉落时不能传送."
+    locked: "&c该洞穴已禁止访客访问!"
+    protected: "&c没有权限: [description]"
+    spawn-protected: "&c没有权限: [description]"
+
     panel:
       PROTECTION:
         description: |-
-          ＆a保护设置
-          ＆a这个洞
+          &a洞穴的
+          &a保护设置
       SETTING:
         description: |-
-          ＆a常规设置
-          ＆a这个洞
+          &a洞穴的
+          &a一般设置
+
 protection:
   flags:
     SKY_WALKER_FLAG:
-      hint: 允许在洞穴顶上行走。
-      name: 天空行者
-      description: |-
-        ＆5＆o这允许启用/禁用
-        ＆5＆o能力继续前进
-        ＆5＆otop洞穴世界
-        ＆5＆o没有额外的权限。
+      description: "&5&o这将允许/禁止\n&5&o没有额外权限的玩家\n&5&o在洞穴世界的顶部\n&5&o行走."
```

Comment:
Why not follow the original and use ` |-`? just out of curiousity

Reply:
This translation is completely based on the file en-US.yml, and it doesn't have `|-`.

Reply:
> Why not follow the original and use ` |-`? just out of curiousity

This translation is completely based on the file en-US.yml, and it doesn't have `|-`.